### ISSUE TITLE
Latest alembic migration mistakenly modifies 'external'

### ISSUE THREAD

**Issue by phmccarty** [clearlinux/telemetrics-backend#12]
The recently added alembic migration modifies the 'external' field, but it should not, because the type of the field was not altered, and the model.py appears correctly formed.

For reference, see https://github.com/clearlinux/telemetrics-backend/blob/d9d1dd2f90dc0d683da3fc3f54805f91b452d8aa/telemetryui/migrations/versions/466cf2f35d67_.py

**Comment by gabrielbriones** [comment]
I got this error message when running migrations:

```
sqlalchemy.exc.ProgrammingError: (psycopg2.ProgrammingError) index "classification_text_ind" does not exist
 [SQL: '\nDROP INDEX classification_text_ind']
```

**Comment by phmccarty** [comment]
@gabrielbriones Yes, I think that's also a bug. I opened #13.

**Comment by phmccarty** [comment]
Resolved with #14
